src/apps/core/i18n/en-US.json

```diff
@@ -1056,5 +1056,15 @@
 		"addRow": "+ Add Data Row",
 		"keyPlaceholder": "Data Key",
 		"valuePlaceholder": "Data Value"
+	},
+
+	"monsterLanguages": {
```

Comment:
You should add this new entry for other languages whenever possible, I am sure you can find the corresponding translations under `core.i18n.languages`.